git.html.markdown

```diff
@@ -79,6 +79,11 @@ this pointer will automatically update and point to the latest commit.
 HEAD is a pointer that points to the current branch. A repository only has 1 *active* HEAD.
 head is a pointer that points to any commit. A repository can have any number of heads.
 
+###Stages of Git
```

Comment:
You're right, it should be modified -> staged -> committed. Must have had a brain fart while writing it.